### PR TITLE
FT Professional

### PR DESCRIPTION
**Pull request by notlee**

### CODE REVIEW

File: components/o-forms/stories/checkboxes.stories.tsx [hunk 1]
```diff
@@ -1,25 +1,42 @@
...
 const hideArg = {
 	table: {
 		disable: true,
 	},
 };
 
+const Brand = process.env.ORIGAMI_STORYBOOK_BRAND;
+const themeControl =
+	Brand === "core"
+		? {
```
Comment (frshwtr) [inline components/o-forms/stories/checkboxes.stories.tsx:18]: interesting indentation here. Is this prettier misbehaving?

File: components/o-buttons/origami.json [hunk 2]
```diff
@@ -76,6 +76,25 @@
...
+			"name": "Professional",
+			"title": "Professional theme",
+			"brands": [
+				"core"
+			],
+			"template": "/demos/src/professional.mustache",
+			"description": "Professional themed buttons."
+		},
+		{
+			"name": "Inverse Professional",
```
Comment (frshwtr) [inline components/o-buttons/origami.json:89]: The naming of these two new demos appear to be inconsistent with the rest of the demos. Could these be update to kebab case?

**Review by frshwtr — APPROVED** [review]

File: components/o-buttons/origami.json [hunk 3]
```diff
@@ -76,6 +76,25 @@
...
+			"title": "Inverse Professional theme",
+			"brands": [
+				"core"
+			],
+			"bodyClasses": "demo-inverse",
+			"template": "/demos/src/professional-inverse.mustache",
+			"description": "Inverse Professional themed buttons."
```
Comment (notlee) [inline components/o-buttons/origami.json:96]:
```suggestion
		{
			"name": "professional",
			"title": "Professional theme",
			"brands": [
				"core"
			],
			"template": "/demos/src/professional.mustache",
			"description": "Professional themed buttons."
		},
		{
			"name": "professional-inverse",
			"title": "Professional Inverse theme",
			"brands": [
				"core"
			],
			"bodyClasses": "demo-inverse",
			"template": "/demos/src/professional-inverse.mustache",
			"description": "Inverse Professional themed buttons."
```

File: components/o-forms/stories/checkboxes.stories.tsx (hunk 1 above)
Comment (notlee) [reply components/o-forms/stories/checkboxes.stories.tsx:18]: Yeah I imagine slightly weird Prettier behaviour

**Pull request merged**